server/routes/authed.js

```diff
@@ -0,0 +1,29 @@
+//Third party packeges
+const express = require('express');
+const router = express.Router();
+
+const isAuth = (req, res, next) => {
+    try {
+        if (!req.session.authorization) res.sendFile(path.join(__dirname, "..", "client/build", "index.html"));
+        else next();
+    } catch (error) {
+        return res.send({ status: true, message: 'your session is not valid', data: error })
+    }
+}
+
+router.post('/authed', isAuth, (req, res) => {
```

Comment:
No need this if you will have a /profile route. It should be called every time users enters the app.